app/controllers/patients_controller.rb

```diff
@@ -453,7 +453,7 @@ def update_status
     # NOTE: This is a possible option when changing monitoring status of HoH in isolation.
     if params.permit(:apply_to_household_cm_exp_only)[:apply_to_household_cm_exp_only] && params[:apply_to_household_cm_exp_only_date].present?
       # Only update dependents (not including the HoH) in exposure with continuous exposure is turned on
-      (current_user.get_patient(patient.responder_id)&.dependents_exclude_self&.where(continuous_exposure: true, isolation: false) || []).uniq.each do |member|
+      (current_user.get_patient(patient.responder_id)&.household&.where(continuous_exposure: true, isolation: false) || []).uniq.each do |member|
```

Comment:
can we just also update the comment on line 453 to say:
```
# NOTE: This is a possible option when changing monitoring status of HoH or dependent in isolation.
```

Reply:
good catch, I updated that comment and the one below it

Reply:
awesome, thank you!